CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+### Fixed
+- Fixed race condition during tests parallel execution (`tests::ingest::test_fetch::*` test group)
```

Comment:
Normally I wouldn't mention non customer facing changes like these in the changelog.